audioread/ffdec.py

```diff
@@ -231,9 +231,9 @@ def _parse_info(self, s):
             if mode == 'stereo':
                 self.channels = 2
             else:
-                match = re.match(r'(\d+) ', mode)
-                if match:
-                    self.channels = int(match.group(1))
+                cmatch = re.match(r'(\d).?(\d)?', mode)
```

Comment:
Oh, good catch! I forgot to re-add the `+` after testing.

Also, the `.` should be escaped. Should be fixed now. The previous version was working on files up to 999, since the center digit was also matched with `.`. Now tested on files with 3  channels (2.1), 6 channels (5.1), and 12256 channels, which is ... unlikely. :-)